release_notes.md

```diff
@@ -1,3 +1,54 @@
+**********************************
+# Release Notes for OpenMDAO 3.7.0
+
+February 11, 2021
+
+OpenMDAO 3.7.0 adds the ability for users to use their own Python functions
+from within ExecComp.  It also adds to OpenMDAO's utility in notebook environment
+by formatting output of the informative methods (list_outputs, view_connections, etc)
+in a notebook-friendly table format.  Other changes include performance improvements
+when using partial derivative coloring, continued improvements to the N2 viewer,
+and removal of the mock dependency
+
+## Backwards Incompatible API Changes:
+
+- None
+
+## Backwards Incompatible NON-API Changes:
+
+- If case recorder outputs are returned scaled, this no longer (erroneously) remains true on subsequent calls. [#1850](https://github.com/OpenMDAO/OpenMDAO/pull/1850)
+
+## New Features:
+
+- ExecComp expressions are visible in the N2 Node Info Panel [#1888](https://github.com/OpenMDAO/OpenMDAO/pull/1888)
+- Added ability to load N2 with specific case [#1883](https://github.com/OpenMDAO/OpenMDAO/pull/1883)
+- [POEM 036](https://github.com/OpenMDAO/POEMs/blob/master/POEM_039.md): User created functions can now be used with ExecComp. ExecComp now allows `shape_by_conn` and `copy_shape` options for IO. [#1852](https://github.com/OpenMDAO/OpenMDAO/pull/1852)
+- Options for all solvers are always recorded [#1845](https://github.com/OpenMDAO/OpenMDAO/pull/1845)
+- Added detection of notebook env, table formatting and in cell HTML output [#1844](https://github.com/OpenMDAO/OpenMDAO/pull/1844)
+- Added a display to the N2 so that if an error occurs the user some idea of what happened [#1838](https://github.com/OpenMDAO/OpenMDAO/pull/1838)
+
+## Bug Fixes:
+
+- Fixed N2 connections toolbar show/hide buttons [#1890](https://github.com/OpenMDAO/OpenMDAO/pull/1890)
+- Fix for the way AnalysisError is raised in parallel. [#1878](https://github.com/OpenMDAO/OpenMDAO/pull/1878)
```

Comment:
Change to "Fixed a bug in nonlinear and linear solvers when running under MPI so that AnalysisErrors for non-convergence are raised on all processors instead of just root."